tests/metrics/test_absolute_volume_difference.py

```
@@ -0,0 +1,160 @@
+        agg = metric.aggregate()
+        self.assertAlmostEqual(agg.item(), 2.0)
+        metric.reset()
+
+    def test_reset_clears_buffer(self):
+        metric = AbsoluteVolumeDifferenceMetric(ignore_empty=False)
+        y = torch.zeros(1, 2, 4)
+        y[0, 1, :2] = 1.0
+        metric(y, y)
+        metric.reset()
+        # after reset the buffer should be empty; calling aggregate raises
+        with self.assertRaises(Exception):
```

Ruff (0.15.18)

[warning] Do not assert blind exception: `Exception`

(B017)